hack/pin-deps.sh

```diff
@@ -57,8 +57,8 @@ edit_args=(
 )
 
 for repo in ${STAGING_REPOS[@]}; do
-    if version_ge "1.17.0" $VERSION; then
-        echo ">=1.17.0"
+    if version_ge "1.16.3" $VERSION; then
```

Comment:
Why change this?

Reply:
The old script import libs with version `$repo@kubernetes-$VERSION`  before v1.17.0. But libs already have semantic versions which can be imported after v1.16.3.